src/Symfony/Component/Asset/VersionStrategy/JsonManifestVersionStrategy.php

```diff
@@ -54,7 +54,7 @@ private function getManifestPath(string $path): ?string
     {
         if (null === $this->manifestData) {
             if (!file_exists($this->manifestPath)) {
-                throw new \RuntimeException(sprintf('Asset manifest file "%s" does not exist.', $this->manifestPath));
+                throw new \RuntimeException(sprintf('Asset manifest file "%s" does not exist. Did you forget to build the assets with npm or yarn?', $this->manifestPath));
```

Comment:
I think the npm and yarn are to much
Build the frontend assets could be enough imho

Reply:
I'm not sure because if we only display this --> _"Did you forget to build the assets?"_ Many newcomers might think this is a problem to be solved in Symfony ... if we add the brief mention to "yarn" and "npm", they'll get an additional clue about how to solve this or how to better look for a solution for this.